tests/ui/union/unnamed-fields/repr_check.rs

```diff
@@ -0,0 +1,69 @@
+#![allow(incomplete_features)]
+#![feature(unnamed_fields)]
+
+struct A { //~ ERROR struct with unnamed fields must have `#[repr(C)]` representation
+           //~^ NOTE struct `A` defined here
```

Comment:
I'm afraid there might be a bug here: I used a label but the UI test harness complained an unexpected note.